### PR TITLE
Add board list and delete to labyrinth admin

### PR DESCRIPTION
**Pull request by jonathan-marmelab**
Depends on #38 

Peek 15-03-2023 10-59

### CODE REVIEW

File: admin/src/board/list.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { List, Datagrid, TextField, DeleteButton } from "react-admin";
+
+export const BoardList = () => (
+  <List>
+    <Datagrid rowClick="edit">
```
Comment (JulienMattiussi) [inline admin/src/board/list.tsx:5]: not sure edit will work if you didn't defined an edit page

File: admin/src/user/list.tsx [hunk 2]
```diff
@@ -18,6 +13,13 @@ export const UserList = () => (
       <TextField source="id" />
       <TextField source="username" />
       <EmailField source="email" />
+      <CustomManyCount
```
Comment (JulienMattiussi) [inline admin/src/user/list.tsx:16]: not fan of the name, I don't understand what this component will display/act on

File: admin/src/shared/CustomManyCount.tsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    perPage: 1,
+    record,
+    reference,
+    resource,
+    source,
+    target,
+    sort: { field: sortBy, order: sortByOrder },
+  });
+
+  const body = isLoading ? (
```
Comment (JulienMattiussi) [inline admin/src/shared/CustomManyCount.tsx:48]: not fan of the embedded ternary. can you flatten this the readibility ?
also maybe inside a <Body> component using only if for each cases, or anything better you can imagine

File: admin/src/shared/CustomManyCount.tsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      ""
+    )
+  ) : error ? (
+    <ErrorIcon color="error" fontSize="small" titleAccess="error" />
+  ) : (
+    total
+  );
+
+  return link ? (
+    // @ts-ignore TypeScript complains that the props for <a> aren't the same as for <span>
```
Comment (JulienMattiussi) [inline admin/src/shared/CustomManyCount.tsx:61]: so why using span ?

**Review by JulienMattiussi — CHANGES_REQUESTED** [review]

File: admin/src/user/list.tsx (hunk 2 above)
Comment (jonathan-marmelab) [reply admin/src/user/list.tsx:16]: This will be resolved in a future PR (I will need to have a custom join table and this component will be removed).

**Review by JulienMattiussi — APPROVED** [review]

**Pull request merged**